src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -130,33 +128,12 @@ class UnifiedShareModal extends React.Component<USMProps, State> {
     };
 
     renderUSF = () => {
-        const {
-            onRemoveAllRestrictedCollabs,
-            // $FlowFixMe - For temporary backwards compatibility
-            onRemoveAllRestrictedExternalCollabs,
-            restrictedCollabEmails,
-            // $FlowFixMe - For temporary backwards compatibility
-            restrictedExternalCollabEmails,
-            sharedLinkEditTagTargetingApi,
-            sharedLinkEditTooltipTargetingApi,
-        } = this.props;
+        const { sharedLinkEditTagTargetingApi, sharedLinkEditTooltipTargetingApi } = this.props;
         const { isFetching, sharedLinkLoaded, shouldRenderFTUXTooltip } = this.state;
 
-        // TODO
-        // For temporary backwards compatibility. Remove legacy onRemoveAllRestrictedExternalCollabs
-        // and restrictedExternalCollabEmails props once EUA has been updated not to pass them
-        const onRemoveAllRestrictedCollabsValue = isNil(onRemoveAllRestrictedExternalCollabs)
-            ? onRemoveAllRestrictedCollabs
-            : onRemoveAllRestrictedExternalCollabs;
-        const restrictedCollabEmailsValue = isEmpty(restrictedExternalCollabEmails)
-            ? restrictedCollabEmails
-            : restrictedExternalCollabEmails;
-
         return (
             <UnifiedShareForm
                 {...this.props}
-                onRemoveAllRestrictedCollabs={onRemoveAllRestrictedCollabsValue}
-                restrictedCollabEmails={restrictedCollabEmailsValue}
```

Comment:
Q: are these still needed within UnifiedShareForm ?

Reply:
Yes, but they're being included with the `{...this.props}` line.